spec/support/mock_time.rb

```diff
@@ -1,15 +1,21 @@
 # frozen_string_literal: true
 
 RSpec.configure do |config|
-  config.around :each, mock_time: true do |example|
+  config.before :each, mock_time: true do
     @mocked_date = Time.utc(1992, 1, 1)
+    @mocked_clock = 123_456_000_000
 
     def travel(distance)
```

Comment:
`travel_micros`?
Does the agent use any other time measurements? If not, then I think it's ok to leave the name as `travel`, as all time units will be assumed to be microseconds. If there are other time references using something other than microseconds, we might want to be more explicit about what time unit this method assumes.

Reply:
Yes, something like that. It's all μs internally until we serialise and convert to ms.